perfkitbenchmarker/virtual_machine.py

```diff
@@ -149,6 +150,42 @@ def CreateScratchDisk(self, disk_spec):
     """
     pass
 
+  def _CreateScratchDiskFromDisks(self, disk_spec, disks):
+    """Helper method to prepare data disks.
+
+    Given a list of BaseDisk objects representing a single data disk,
```

Comment:
We support benchmarks that use more than one "logical" data disk like the copy_throughput benchmark (which copies data from one scratch disk to another). I meant that all of the BaseDisk objects passed to this function correspond to a single logical data disk in that they will be treated as a single disk from the benchmarks perspective. Changed the docstring to make this more clear.